src/SB/Game/zNPCTypeTiki.h

```diff
@@ -50,19 +46,61 @@ struct zNPCTiki : zNPCCommon
     void ParseINI();
     void SelfSetup();
     S32 IsHealthy();
-    virtual S32 IsAlive();
-    U8 PhysicsFlags() const;
-    U8 ColChkByFlags() const;
-    U8 ColPenFlags() const;
-    U8 ColChkFlags() const;
-    U8 ColPenByFlags() const;
     virtual void Damage(en_NPC_DAMAGE_TYPE damtype, xBase* who, const xVec3* vec_hit);
-    virtual S32 Respawn(xVec3* pos, zMovePoint* mvptFirst, zMovePoint* mvptSpawnRef);
-    virtual void DuploOwner(zNPCCommon* duper);
-    virtual void DuploNotice();
-    virtual S32 CanRope();
     S32 SetCarryState(en_NPC_CARRY_STATE cs);
-    virtual U32 AnimPick(int, en_NPC_GOAL_SPOT, xGoal*);
+
+    zNPCTiki(S32 myType) : zNPCCommon(myType)
+    {
+    }
+
+    virtual S32 CanRope()
+    {
+        return 1;
+    }
+
+    virtual U32 AnimPick(int, en_NPC_GOAL_SPOT, xGoal*)
+    {
+        return xStrHash("Idle");
+    }
+
+    virtual void Move(xScene* xscn, F32 dt, xEntFrame*)
+    {
+    }
+
+    virtual void BUpdate(xVec3* pos)
+    {
+        xEntDefaultBoundUpdate(this, pos);
+    }
+
+    virtual S32 IsAlive()
+    {
+        return (0x200 - (tikiFlag & 0x300) | (tikiFlag & 0x300) - 0x200) >> 0x1f;
+    }
+
+    virtual U8 ColChkFlags() const
+    {
+        return 0;
+    }
+
+    virtual U8 ColPenFlags() const
+    {
+        return 0;
+    }
+
+    virtual U8 ColChkByFlags() const
+    {
+        return 24;
+    }
+
+    virtual U8 ColPenByFlags() const
+    {
+        return 24;
```

Comment:
Return the hexadecimal bit representation here, as it will be more useful for analysis of the function's usages